src/Microsoft.Extensions.FileProviders.Abstractions/NullFileProvider/NullFileProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.Extensions.FileProviders.Abstractions
+{
+    public class NullFileProvider : IFileProvider
```

Comment:
Should this go in `Microsoft.Extensions.FileProviders.Null` seeing that's how the rest of the types here are structured?